docs/user/customization.md

```diff
@@ -234,3 +234,101 @@ An example `worker.ign` is shown below. It has been modified to increase the HTT
 ```
 
 [ignition]: https://coreos.com/ignition/docs/latest/
+
+## NTP
+
+RHCOS hosts will contain a sane default NTP configuration for an internet
```

Comment:
I think it'd be better if this doc lived in the MCO repo and we just linked to it...this exact same example is [in the MCO repo](https://github.com/openshift/machine-config-operator/#applying-configuration-changes-to-the-cluster).

(Or of course...in the OpenShift docs)

Reply:
BTW see also https://github.com/openshift/machine-config-operator/issues/629